formats/polus-ome-zarr-converter-plugin/VERSION

```diff
@@ -1 +1 @@
-0.2.0
\ No newline at end of file
+0.5.0
```

Comment:
Why are we jumping over 2 minor versions?

I'm okay with this for now, but please review semantic versioning.
https://semver.org/

In general, adding new functionality should bump the minor version as long as you don't break anything. I generally don't like skipping minor versions unless there is a good reason.

Reply:
Main reason for jumping over 2 minor versions bz I registered plugin on WIPP and there was an issue with plugin manifest. I renamed the plugin as you suggestion to polus-ome-converter-plugin and started with 0.1.0 version